docker/test_env/geth.go

```diff
@@ -164,15 +169,16 @@ func (g *Geth) getGethContainerRequest(networks []string) (*tc.ContainerRequest,
 		Name:            g.ContainerName,
 		AlwaysPullImage: true,
 		Image:           "ethereum/client-go:stable",
-		ExposedPorts:    []string{"8544/tcp", "8545/tcp"},
+		ExposedPorts:    []string{exposedHttpPort, exposedWsPort},
 		Networks:        networks,
 		WaitingFor: tcwait.ForAll(
 			tcwait.NewHTTPStrategy("/").
-				WithPort("8544/tcp"),
+				WithPort(exposedHttpPort),
 			tcwait.ForLog("WebSocket enabled"),
 			tcwait.ForLog("Started P2P networking").
 				WithStartupTimeout(120*time.Second).
 				WithPollInterval(1*time.Second),
+			NewWebSocketStrategy(),
```

Comment:
please add this similar strategy for non-dev-geth as well

Reply:
Also a good call, done.